client/post-editor/editor-featured-image/preview-container.jsx

```diff
@@ -5,17 +5,16 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import { defer } from 'lodash';
+import { connect } from 'react-redux';
 
 /**
  * Internal dependencies
  */
-import MediaActions from 'lib/media/actions';
 import MediaStore from 'lib/media/store';
 import EditorFeaturedImagePreview from './preview';
+import { fetch } from 'state/media/thunks';
```

Comment:
FWIW I liked when you aliased it as `fetchMediaItem`, I think it made the consuming code clearer 🙂

Reply:
Good catch 👍 I think renaming the export makes sense then.